tasks/managed/filter-already-released-advisory-images/filter-already-released-advisory-images.yaml

```diff
@@ -212,6 +216,23 @@ spec:
           advisorySecretName="create-advisory-stage-secret"
         fi
 
+        # Check if skipFilter is true in data.json
+        DATA_FILE="$(params.dataDir)/$(params.dataPath)"
+        SKIP_FILTER="false"
+        if [ -f "$DATA_FILE" ]; then
```

Comment:
This seems overly complex. For one, you know we will have a data file. If you want to protect against it not being there, sure. If the data file exists and the `jq -r '.skipFilter` is true, then do your echoes and exit. There is no need to have a default in the jq query or so many ifs IMO